client/clientservice/include/client/clientservice/state_snapshot_service.hpp

```diff
@@ -37,6 +37,11 @@ class StateSnapshotServiceImpl final
                         vmware::concord::client::statesnapshot::v1::ReadAsOfResponse* response) override;
 
  private:
+  bool isHashValid(uint64_t snapshot_id,
+                   concord::util::SHA3_256::Digest& final_hash,
```

Comment:
Can we make `final_hash` a const ref?

Reply:
Thanks @dartdart26 , I will add this in next commit.